config/vendor_libraries.cmake

```diff
@@ -191,6 +201,7 @@ macro( setupLAPACKLibrariesUnix )
         IMPORTED_LINK_INTERFACE_LIBRARIES_DEBUG
         IMPORTED_LINK_INTERFACE_LIBRARIES_RELEASE
         IMPORTED_LINK_INTERFACE_LIBRARIES_RELWITHDEBINFO
+        IMPORTED_LOCATION_RELWITHDEBINFO
```

Comment:
This is a mistake.  I'll fix it and sort the properties so duplicates are easier to see.